cloudant-sync-datastore-core/src/main/java/com/cloudant/sync/query/Tokenizer.java

```diff
@@ -0,0 +1,76 @@
+package com.cloudant.sync.query;
+
+/**
+ * Specify the SQLite FTS
+ * <a target="_blank" href="https://www.sqlite.org/fts3.html#creating_and_destroying_fts_tables">
+ * tokenizer</a> to use for indexing.
```

Comment:
Given that the link word is `tokenizer` I think https://www.sqlite.org/fts3.html#tokenizer maybe a more appropriate destination, although I think both could be relevant.